lib/pacemaker/pcmk_sched_notif.c

```diff
@@ -560,7 +560,7 @@ expand_notification_data(pe_resource_t *rsc, notify_data_t * n_data, pe_working_
 
     if (n_data->start) {
         n_data->start = g_list_sort(n_data->start, sort_notify_entries);
-        if (rsc_list && safe_str_eq(n_data->action, RSC_START)) {
+        if (rsc_list && pcmk__str_eq(n_data->action, RSC_START, pcmk__str_casei)) {
```

Comment:
Line 558 sets rsc_list to NULL, so this condition will always fail. (Thanks, Coverity!)

Reply:
Crap. Looks like a regression in 4974f082 (2.0.1). Thankfully this is the only place in this function with this issue. I'll do a fix in a separate PR.

Reply:
It turns out this is meaningless. The condition is only used to change the return value, and the return value is ignored. I'll just change it to a void function instead. At least there's no regression :)

Reply:
I spoke too soon. It is used internally to decide whether to make the pre-/post-notifications required. I'll have to figure out what impact that actually has.